patches/server/0752-Fix-endermen-shulkers-tp-ing-above-logical-height.patch

```diff
@@ -0,0 +1,81 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Rektroth <53879295+Rektroth@users.noreply.github.com>
+Date: Sun, 22 Aug 2021 23:59:54 -0400
+Subject: [PATCH] Fix endermen/shulkers tp'ing above logical height
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/LivingEntity.java b/src/main/java/net/minecraft/world/entity/LivingEntity.java
+index 6175360eb2b19c8197cc5b82a09030211afd838b..23ac83913d41af11d40618661e9d478b0caf7836 100644
+--- a/src/main/java/net/minecraft/world/entity/LivingEntity.java
++++ b/src/main/java/net/minecraft/world/entity/LivingEntity.java
+@@ -3899,7 +3899,15 @@ public abstract class LivingEntity extends Entity {
+                 // first set position, to check if the place to teleport is valid
+                 this.setPos(d0, d6, d2);
+                 if (world.noCollision(this) && !world.containsAnyLiquid(this.getBoundingBox())) {
++<<<<<<< Updated upstream
+                     flag1 = true;
++=======
++                    // Paper start - check if the place to teleport is above the dimension's logical height
++                    if (d6 < world.dimensionType().logicalHeight()) {
++                        flag1 = true;
++                    }
++                    // Paper end
++>>>>>>> Stashed changes
```

Comment:
You've got some leftover git merge stuff

Reply:
Deleted.